robocop/checkers/__init__.py

```diff
@@ -61,6 +60,26 @@ def report(self, msg, *args, node=None, lineno=None, col=None):
 
     def configure(self, param, value):
         self.__dict__[param] = value
+
+
+class VisitorChecker(BaseChecker, ast.NodeVisitor):
+    type = 'visitor_checker'
+
+    def visit_File(self, node):
+        self.generic_visit(node)
+
+
+class RawFileChecker(BaseChecker):
+    type = 'rawfile_checker'
+
+    def visit_file(self):
```

Comment:
Inconsistent case in model naming. sometimes you use capitals, and sometimes lowercase :)

Reply:
I'm forced to used capitals if I'm implementing checkers based on official robot parsing api. It only matches visit_ClassName methods, where ClassName is title capitalized. I could change method names in RawFileChecker so it would be not confused with VisitorChecker